python/mxnet/contrib/svrg_optimization/README.md

```diff
@@ -0,0 +1,41 @@
+## SVRG Optimization in Python Module
+
+### Problem Description
+SVRG stands for Stochastic Variance Reduced Gradient, which was first introduced in the paper _Accelerating Stochastic 
+Gradient Descent using Predicative Variance Reduction_ in 2013. It is an optimization technique that complements SGD. 
+SGD is known for large scale optimization but it suffers from slow convergence asymptotically due to the inherent 
+variance. SGD approximates the full gradient using a small batch of samples which introduces variance. 
+In order to converge faster, SGD often needs to start with a smaller learning rate. SVRG remedies the problem by keeping 
+a version of the estimated weights that is close to the optimal parameters and maintain average of full gradient over 
+full pass of data. The average of full gradients of all data is calculated w.r.t to parameters of last mth epochs. 
+It has provable guarantees for strongly convex smooth functions, and a more detailed proof can be found in section 3 of 
+the paper. SVRG uses a different update rule: gradients w.r.t current parameters minus gradients w.r.t parameters 
+from the last mth epoch, plus the average of gradients over all data. 
+
+#### Key Characteristics of SVRG
+* Explicit Variance Reduction
+* Ability to use relatively large learning rate compared to SGD, which leads to faster convergence compared to SGD.
+
+#### Testing:
+
+Unit Tests:
+* test_svrg_module.py: unittests for SVRGModule API
+* test_svrg_optimizer.py: unittests for SVRGOptimizer API
+
+### Examples:
+* examples/svrg_module/train.py: test script for training a linear regression model with SVRG optimization on YearPrediction MSD dataset
+* examples/svrg_module/example_api_train.py: a demo script for training a SVRGModule using intermediate level api and high level api
+* examples/svrg_module/example_inference.py: a demo script for SVRGModule inference
+
+#### Benchmarks:
```

Comment:
as the benchmark graphs are present in the "examples" folder, move these lines to the README in the examples folder.